src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -4391,144 +4391,5 @@ public void BaseExpression_01()
             }
             EOF();
         }
-
-        [Fact]
```

Comment:
>Fact [](start = 9, length = 4)

General question about this PR: 
I don't think the removal of `base(Something)` is a breaking change, since the feature has not RTM'ed. But if anyone has been using this feature in preview, their code will break. Does that rise to the level where we want to add a note to breaking changes?

Reply:
The feature became available only a couple of weeks ago in Preview 3 of VS.

---
In reply to: [280927546](https://github.com/dotnet/roslyn/pull/35456#discussion_r280927546) [](ancestors = 280927546)